go/vt/vtctld/api_test.go

```diff
@@ -258,9 +299,9 @@ func TestAPI(t *testing.T) {
 				"sharding_column_name": "shardcol",
 				"sharding_column_type": 0,
 				"served_froms": [],
-                                "keyspace_type":0,
-                                "base_keyspace":"",
-                                "snapshot_time":null
+				"keyspace_type":0,
+				"base_keyspace":"",
+				"snapshot_time":null
```

Comment:
I assume this is tabs vs spaces. I remember struggling with this before 😠

Reply:
Those three lines are now indented using tabs, like the other lines on this file